universal-login-sdk/lib/core/observers/ObserverRunner.ts

```diff
@@ -28,10 +29,11 @@ abstract class ObserverRunner {
 
   stop() {
     this.state = 'stop';
+    clearTimeout(this.timeout);
   }
 
   async finalizeAndStop() {
-    this.state = 'stopping';
+    this.state = this.isRunning() ? 'stopping' : 'stop';
```

Comment:
This is probably a bug, i.e.:
`if (state == 'stopping')` `finalizeAndStop()` will change it to `stop` while it should not.

Reply:
Implementation of isRunnning bellow: 
``` ts 
protected isRunning() {
    return this.state !== 'stop';
 }
```
So if state === 'stopping' finalizeAndStop `isRunning()` will be true, so finalizeAndStop will set state to 'stopping'

Reply:
Ok, so it is a bad getter. IsRunning should returninfo if observer is in state running not running or stopping

Reply:
So I will update it and change condition